### PR TITLE
fix: add stream-chat@8.x.x to peerDependencies

### PR DESCRIPTION
**Pull request by MartinCupela**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
   "peerDependencies": {
     "react": "^18.0.0 || ^17.0.0 || ^16.8.0",
     "react-dom": "^18.0.0 || ^17.0.0 || ^16.8.0",
-    "stream-chat": "^7.1.0"
+    "stream-chat": "^8.0.0 || ^7.1.0"
```
Comment (arnautov-anton) [inline package.json:67]:
```suggestion
    "stream-chat": ">= 7.1.0"
```

Comment (arnautov-anton) [reply package.json:67]: Or whatever oldest version we decide to support.

Comment (oliverlaz) [reply package.json:67]: I think we should stay in the range that @MartinCupela proposed for the time being.
I'm not sure how safe would it be if we have, for example, `chat@10.4` and `client@12`.

**Review by oliverlaz — APPROVED** [review]

**Comment by oliverlaz** [comment]
Fixes: #1813

**Pull request merged**